source/CILoadingScene.cpp

```diff
@@ -103,6 +102,18 @@ void LoadingScene::dispose() {
  * @param timestep  The amount of time (in seconds) since the last frame
  */
 void LoadingScene::update(float progress) {
+    if (isActive()) { // initially set all values to default
```

Comment:
So I just tested and the stuff in if (isActive()) is not necessary so please delete it